great_expectations/experimental/metric_repository/column_descriptive_metrics_metric_retriever.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Sequence
+
+from great_expectations.experimental.metric_repository.metric_retriever import (
+    MetricRetriever,
+)
+from great_expectations.experimental.metric_repository.metrics import (
+    Metric,
+    MetricException,
+    NumericTableMetric,
+    StringListTableMetric,
+)
+from great_expectations.validator.metric_configuration import MetricConfiguration
+
+if TYPE_CHECKING:
+    from great_expectations.datasource.fluent import BatchRequest
+
+
+class ColumnDescriptiveMetricsMetricRetriever(MetricRetriever):
+    # TODO: Docstrings
+    def get_metrics(self, batch_request: BatchRequest) -> Sequence[Metric]:
+        table_metrics_list = self._get_table_metrics(batch_request)
+
+        return table_metrics_list
+
+    def _get_table_metrics(self, batch_request: BatchRequest) -> Sequence[Metric]:
+        table_metric_names = ["table.row_count", "table.columns"]
+        table_metric_configs = [
+            MetricConfiguration(
+                metric_name=metric_name, metric_domain_kwargs={}, metric_value_kwargs={}
+            )
+            for metric_name in table_metric_names
+        ]
+        validator = self._context.get_validator(batch_request=batch_request)
+        computed_metrics = validator.compute_metrics(
+            metric_configurations=table_metric_configs,
+            runtime_configuration={"catch_exceptions": True},  # TODO: Is this needed?
```

Comment:
Intent to keep the `catch_exceptions=True` is so that the Exceptions information can be written into the database, and we have a record / know where to go for debugging in front end. There may still be a bug here, but we will get this parameter to work.